install

```diff
@@ -0,0 +1,28 @@
+-- Install the program by rewriting dofiles, and creating a link to the program
+
+local dest_dir = shell.resolve(select(..., 1) or '_build')
```

Comment:
what do you mean by killing the installation?  You can delete a program's file while it's running and CC will continue to run the program as `temp`